internal/venice-common/src/test/java/com/linkedin/venice/helix/HelixReadOnlySchemaRepositoryTest.java

```diff
@@ -157,8 +158,8 @@ public void testForceRefreshSchemaData() {
     Assert.assertNull(schemaRepository.getSupersetSchema(storeName));
   }
 
-  @Test
-  public void getDerivedSchemaIdTest() {
+  @Test(dataProviderClass = DataProviderUtils.class, dataProvider = "Two-True-and-False")
```

Comment:
Should we have assertions to check whether the code enabled by `wcEnabled` & `aaEnabled` was called or not?

Reply:
Thanks